addOns/exim/src/main/java/org/zaproxy/addon/exim/ImportExportApi.java

```diff
@@ -63,6 +67,10 @@ public ApiResponse handleApiAction(String name, JSONObject params) throws ApiExc
                 } else {
                     throw new ApiException(Type.BAD_EXTERNAL_DATA, file.getAbsolutePath());
                 }
+            case ACTION_IMPORTURLS:
+                UrlsImporter.importUrlFile(
+                        new File(ApiUtils.getNonEmptyStringParam(params, PARAM_FILE_PATH)));
+                return ApiResponseElement.OK;
```

Comment:
It would be better to return the errors that might happen during the import.

Reply:
Agreed. I need to do similar handling for the Log import migration as well. Mind if I put this off till another PR? (I can add a task for it in the tracker.)

Reply:
Sounds good.